### ISSUE TITLE
Our program should be more diverse

### ISSUE THREAD

**Issue by astruxie** [EHanz/PackageStack#5]
I think to maximize our efficiency and achieve most lightweightness, we should have at least be working with more than 5 languages at most. i really think this will eliminate our overhead and keep everything straightforward.

**Comment by jkcarney** [comment]
Good call. I'm thinking C and C++, javascript, typescript, python, C#, ruby on rails, F#, common lisp, and rust. anything less is preposterous

**Comment by mikewilzn** [comment]
We want our code to be very functional as well so we ought to include some OCaml and Lisp.

**Comment by anOrgandroiD** [comment]
No no no, we need to use esoteric programming languages, as copyright infringement is something the team takes very seriously, and these languages are nearly impossible to parse. I recommend we incorporate Whitespace, LOLCODE, and Brainf*ck into Package Stack's design.
